src/main/java/be/woutschoovaerts/mollie/data/order/OrderRequest.java

```diff
@@ -51,7 +52,7 @@
 
     private Map<String, Object> metadata;
 
-    private Optional<Date> expiresAt = Optional.empty();
+    private Optional<LocalDate> expiresAt = Optional.empty();
```

Comment:
Every where else in the application we are using the old Date. I prefer no to mix them at the moment.

Could you refactor your code to use Date instead?

Reply:
shall we take this moment to refactor the entire sdk to use LocalDate/LocalDateTime? Date is almost Deprecated...

Reply:
That is indeed the best solution, but currently I don't have the time, so If one of you guys want to go at it, I'll review the PR :)

Reply:
Sorry my reply took this long time - but I have changed the current part to work with java Dates - in the meantime I started a separate branch for the java time classes :)